db/schema.rb

```diff
@@ -1389,7 +1394,6 @@
     t.boolean "control"
     t.boolean "empty_supplier_sample_name", default: false
     t.boolean "updated_by_manifest", default: false
-    t.boolean "migrated_consent_withdrawn_to_metadata", default: false, null: false
```

Comment:
Bit weird that this is still hanging around. Seems legit to remove it though as it's not there on the table in prod.